pyroute2/netlink/rtnl/__init__.py

```diff
@@ -120,16 +120,16 @@
 TC_H_ROOT = 0xffffffff
 
 
-RTNL_GROUPS = RTNLGRP_IPV4_IFADDR |\
-    RTNLGRP_IPV6_IFADDR |\
-    RTNLGRP_IPV4_ROUTE |\
-    RTNLGRP_IPV6_ROUTE |\
-    RTNLGRP_IPV4_RULE |\
-    RTNLGRP_IPV6_RULE |\
-    RTNLGRP_NEIGH |\
-    RTNLGRP_LINK |\
-    RTNLGRP_TC |\
-    RTNLGRP_MPLS_ROUTE
+RTMGRP_DEFAULTS = RTMGRP_IPV4_IFADDR |\
```

Comment:
Found another issue. On the line 116 I map all the variables that start with `RTM` to create a dictionary.

It wasn't a problem with `RTNLGRP`, but with `RTMGRP` all the groups become mapped too. Could you please change on the line 116
```
map_namespace('RTM', globals())
```
to
```
map_namespace('RTM_', globals())
```
so it will not catch the groups?

thanks

Reply:
Oh sure, I missed that.